Remotion/Web/Core/Res/Themes/NovaGray/HTML/TabStrip.css

```diff
@@ -142,6 +142,11 @@ body.msie *.tabStripTabSelected a::before
   vertical-align: middle;
 }
 
+*.tabStripTabsPane *.anchorBody > img + span
+{
+  margin-left: 3px;
```

Comment:
please use `em` here. white space width increases with font-size, so we need to use `em` instead of `px`.

I'm guessing 0.3em should do the trick, but please test it. it doesn't have to be pixel-perfect but it should be a good match and I don't want to go into milli-fractions